src/plugins/discover/public/application/main/components/layout/discover_documents.tsx

```diff
@@ -140,15 +140,19 @@ function DiscoverDocumentsComponent({
 
   const expandedDoc = useInternalStateSelector((state) => state.expandedDoc);
 
+  const isTextBasedQuery = useMemo(() => getRawRecordType(query) === RecordRawType.PLAIN, [query]);
```

Comment:
Tbh I think we should delete `isTextBasedQuery` discover wrapper and use `isOfAggregateQueryType` directly instead.

Reply:
@davismcphee 

* forgot to add the mention in the comment above

Reply:
That would work for me too, my main concern is really just around consistency in the utils we use. Personally I find `isOfAggregateQueryType` harder to grok than `isTextBasedQuery` since `aggregate query` is an unclear term to me, but that's more of a nit than anything as long as we're consistent.

Reply:
Something for a later cleanup. I would like minimize extra refactoring shortly before FF as it's unrelated to the current ticket.